pytube/streams.py

```diff
@@ -334,8 +341,10 @@ def download(
                     # send to the on_progress callback.
                     self.on_progress(chunk, fh, bytes_remaining)
         self.on_complete(file_path)
+        self.stop_signal = False        
```

Comment:
Why is this set again here?

Reply:
In case you want to download again using the same Stream object. Otherwise it wouldn't run